compiler/il/OMRBlock.cpp

```diff
@@ -1907,7 +1907,11 @@ OMR::Block::StandardException OMR::Block::_standardExceptions[] =
    {19, "ArithmeticException", CanCatchDivCheck },
    {19, "ArrayStoreException", CanCatchArrayStoreCheck },
    {19, "VirtualMachineError", CanCatchNew | CanCatchArrayNew },
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+   {20, "NullPointerException", CanCatchNullCheck | CanCatchArrayStoreCheck },
```

Comment:
It isn't guaranteed to catch an ArrayStoreCheck though so some care is gong to be needed - people may assume it must rather than it can so you'll need to double check usage.

Reply:
Thanks for catching that!  I'll review how the distinction between "can" and "must" is drawn.

Reply:
Looking at the usage of these flags, my understanding is that they are treated as indicating that a catch block with that particular kind of `Throwable` might catch an exception thrown by that particular kind of check, but not that it necessarily will.